decidim-debates/app/presenters/decidim/debates/debate_presenter.rb

```diff
@@ -26,19 +26,16 @@ def author
                     end
       end
 
-      def title(links: false, all_locales: false)
+      def title(links: false, html_escape: false, all_locales: false)
```

Comment:
💬  In order to prevent breaking things, I'd try to keep the previous behaviour on all unchanged code. 

I understand that, before this change, all the titles were html escaped by default, right? If it is like that, does it make sense to use `true` for the default value, and change it to `false` only on the places where it should be changed? How many changes would you have to do in that case?

On the other hand, I can't find a reason to add the parameter in the middle on the argument list, would it be possible to add it at the end?

Reply:
From what we have noticed, the solution is to not escape it by default, as upon form editing is already passing the model through escaping, so the entities will be exposed to end user within forms. 

You are right with the order of the parameters, however, MeetingPresenter and ProposalPresenter are using the second argument for html escaping, therefore, this is only an alignment with all the other presenters.